src/components/LabelWrapper/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react"
+import { connect } from "react-redux"
+import styles from "./styles.module.css"
+import LabelMenus from "../LabelMenus/index"
+import LabelTitle from "../LabelTitle/index"
+import Maintenance from "../_Labels_/Maintenance/index"
+import Overview from "../_Labels_/Overview/index"
+import UseCases from "../_Labels_/UseCases/index"
+
+class LabelWrapper extends Component {
+  constructor() {
+    super()
+    this.state = {}
+  }
+
+  render() {
+    return (
+      <div className={styles.labelWrapper}>
+        <LabelTitle />
+        <div className={styles.flexComponents}>
+          <LabelMenus />
+          {this.props.base === "Overview" ? (
+            <Overview />
+          ) : this.props.base === "Use Cases and Alerts" ? (
+            <UseCases />
+          ) : this.props.base === "Maintenance" ? (
+            <Maintenance />
+          ) : (
+            <Overview />
+          )}
+        </div>
+      </div>
+    )
+  }
+}
```

Comment:
This is awesome. Can you add `defaultProps` and `propType` declarations to the class as well so that we make it clear what the class should expect, and a default in case the redux store is empty for some unforeseen reason?

Example:
```jsx
import PropTypes from "prop-types"
.
.
.
class LabelWrapper extends Component {
  .
  .
  .
}

LabelWrapper.propTypes = {
  base: PropTypes.string.isRequired
}

LabelWrapper.defaultProps = {
  base: "Overview"
}
```